R/find_peptide_in_pdb.R

```diff
@@ -0,0 +1,86 @@
+#' Finds peptide positions in a PDB structure based on positional matching
+#'
+#' Finds peptide positions in a PDB structure. Often positions of peptides in UniProt and a PDB structure are different due to different
+#' lengths of structures. This function maps a peptide based on its UniProt positions onto a PDB structure. This method is superior to
+#' sequence alignment of the peptide to the PDB structure sequence, since it can also match the peptide if there are truncations or
+#' mismatches.This function also provides an easy way to check if a peptide is present in a pdb structure.
+#'
+#' @param peptide_data a data frame containing at least the input columns to this function.
+#' @param peptide_sequence a character column in the \code{peptide_data} data frame that contains the sequence or ID of the peptide that
+#' should be found.
+#' @param start a numeric column in the \code{peptide_data} data frame that contains start positions of peptides.
+#' @param end a numeric column in the \code{peptide_data} data frame that contains end positions of peptides.
+#' @param uniprot_id a character column in the \code{peptide_data} data frame that contains UniProt identifiers that correspond to the
+#' peptides.
+#' @param pdb_data optional data frame containing data obtained with \code{fetch_pdb()}. If not provided information is fetched automatically.
+#' If this function should be run multiple times it is faster to fetch the information once and provide it to the function. If provided,
+#' make sure that the column names are identical to the ones that would be obtained by calling \code{fetch_pdb()}.
+#'
+#' @return A data frame that contains peptide positions in the corresponding PDB structures. If a peptide is not found in any structure,
```

Comment:
Would it not be good to also include the peptides that were not found? 
You could either add a comment e.g. "not found" or just return it as NA